wp-factories.php

```diff
@@ -0,0 +1,3 @@
+<?php
+// Load factories files.
```

Comment:
Should we add a version number is our plugin ?
Should we prevent direct access to the file ?